packages/react-google-maps-api/src/useLoadScript.ts

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react'
+
+import { injectScript } from './utils/injectscript'
+import { preventGoogleFonts } from './utils/prevent-google-fonts'
+
+import { isBrowser } from './utils/isbrowser'
+import { defaultLoadScriptProps } from './LoadScript'
+import invariant from 'invariant';
+
+export interface UseLoadScriptOptions {
+  googleMapsApiKey?: string;
+  googleMapsClientId?: string;
+  id?: string;
+  version?: string;
+  language?: string;
```

Comment:
How do I change language in runtime?

Reply:
Just pass a new value to the hook? It will assemble new url and effect will re-rerun.

Reply:
Btw, in reality, changing language is not something the user does very often. For a fully localized app, it's usually better to just do a full reload to make sure everything is correctly loaded. It's surely a cool effect when it's changed in runtime, but not worth the hassle imo.

Reply:
in reality this is a part of my own app functionality, and it is important for my own project. sadly google doesn't provide an easy api to switch locale in runtime at all. I had this issue with original react-google-maps, and this functionality is one of main reasons why I started this project at first place.

Reply:
Ok, was just saying, but it should work nonetheless. You need to kinda try it, that's the purpose of this Next approach to see if it can work without that messy cleanup.